migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta
+displayName: Upgrade to Axonframework 4.7 Jakarta
+description: This is what you will want if you also upgrade to Spring 6, Spring Boot 3, or another framework that depends on jakarta packages.
+recipeList:
+  # Convert to Spring Boot 3, Spring FrameWork 6, Jakarta packages, JUnit 5, Java 17, etc.
+  #TODO - org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_0
```

Comment:
Wasn't sure about this line; it ought to work; you're welcome to try it on some sample projects.

Reply:
Although Axon works well with Spring, it should by no means be a requirement for our users.

So, I'd rather have a separate yml file combining the Jakarta (so, I assume this file) and SpringBoot3 rewrite tools.
That would allow non-Spring and Spring users to benefit from this.

Reply:
Swapped the dependency to `rewrite-migrate-java` and recipe to `org.openrewrite.java.migrate.jakarta.JavaxMigrationToJakarta`.

Reply:
The documentation can then separately mention a migration of Spring, and defer to the OpenRewrite documentation at:
https://docs.openrewrite.org/reference/recipes/java/spring/boot3/upgradespringboot_3_0

Reply:
Nice, thanks for this adjustment, @timtebeek!